content/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh.md

```diff
@@ -0,0 +1,320 @@
+--set 'meshConfig.defaultConfig.proxyStatsMatcher.inclusionRegexps[0]=.*'
+```
+
+Run above commands on both AWS EKS cluster and private Kubernetes cluster. It installs the *istio* to the namespace *istio-system* of the clusters. It enables explicitly the Envoy Access Log Service (ALS), with both  *envoyAccessLogService.address* and  *envoyMetricsService.address* settings pointing to the Apache SkyWalking OAP server ELB host name and its port *11800*.
+
+We can check the detailed *istio* installation by typing the following *kubectl* command:
+
+```markdown
+$ kubectl get all -n istio-system
+NAME                                        READY   STATUS    RESTARTS   AGE
+pod/istio-egressgateway-78fb5cf46-djxv2     1/1     Running   0          38d
+pod/istio-ingressgateway-77b9d69b74-499vf   1/1     Running   0          38d
+pod/istiod-67fcb675b5-2dhjw                 1/1     Running   0          38d
+
+NAME                           TYPE           CLUSTER-IP      EXTERNAL-IP                                                               PORT(S)                                                                      AGE
+service/istio-egressgateway    ClusterIP      172.20.224.42   <none>                                                                    80/TCP,443/TCP                                                               38d
+service/istio-ingressgateway   LoadBalancer   172.20.249.41   a17641fd9b6564b02ab3cc5faeb51e7a-1241343104.us-east-2.elb.amazonaws.com   15021:30506/TCP,80:31341/TCP,443:31933/TCP,31400:31131/TCP,15443:32118/TCP   38d
+service/istiod                 ClusterIP      172.20.95.232   <none>                                                                    15010/TCP,15012/TCP,443/TCP,15014/TCP                                        38d
+
+NAME                                   READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/istio-egressgateway    1/1     1            1           38d
+deployment.apps/istio-ingressgateway   1/1     1            1           38d
+deployment.apps/istiod                 1/1     1            1           38d
+
+NAME                                              DESIRED   CURRENT   READY   AGE
+replicaset.apps/istio-egressgateway-78fb5cf46     1         1         1       38d
+replicaset.apps/istio-ingressgateway-77b9d69b74   1         1         1       38d
+replicaset.apps/istiod-67fcb675b5                 1         1         1       38d
+```
+
+### Deploy Online Boutique application
+
+Please follow up the deployment process in [the first blog](https://developer.hpe.com/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise/) to deploy the Online Boutique application across the public AWS EKS cluster and the private Kubernetes cluster in HPE GreenLake for Private Cloud Enterprise. The *Skupper* CLI tool is used for deploying the Virtual Application Network (VAN) and creating connection between private Kubernetes cluster and public AWS EKS cluster.
```

Comment:
Please follow up the deployment process in my blog post [here](https://developer.hpe.com/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise/) to deploy....